scripts/deploy.sh

```diff
@@ -7,17 +7,88 @@ pull_latest() {
 
 rebuild_regs() {
   # Build WCA regulations
-  # Use only one core because we get better backtraces without multiprocessing,
-  # and because we might be on the production server and should not eat up all
-  # cores just building the regulations.
-  regulations/make.py --num-workers 1 --wca
-  regs_folder=WcaOnRails/app/views/regulations
+  # Uses wrc, see here: https://github.com/cubing/wca-regulations-compiler
```

Comment:
I think I was envisioning something more like "give me all the views for the regulations", maybe someday the WRC decides to generate some other format, I don't think the WCA website necessarily needs to know about that.

More important than that, though, I'm just bummed that the `rebuild_regs` function is so long. I was kind of hoping it would be be a 1 line call to `wrc`. I don't think we (the software team) need to be aware of the fact that the regulations and the translations are stored in git, or how to "Checkout data" (I would support moving `regulations-data` branch off of the WCA website and into something controlled by the WRC), or even the fact that the regulations have translations =).

That all said, I'm totally fine merging this up as is. We can discuss later what exactly we consider "WRC logic".